### DISCUSSION TITLE
There is outstanding issues with your account when trying to renew student pack

### DISCUSSION THREAD

**Discussion by HorselessName** [community/community#68086] [GitHub Education]
### Hi everyone,

Hello. I am trying to renew my student developer pack but am not being able to do so.
I am seeing a message saying that there is outstanding issues with my account.

I already opened a support ticket and they asked me to come post here...
So, what is happening? :(

**Comment by Irasil** [comment]
Hey I had the same problem and my attempts to renew it were rejected over 40 times. 

Like you, I wrote a ticked message and after 1 month I received an answer that my case was being looked at and it could take up to 72 hours. After that I didn't hear anything again and opened the ticket again and asked every 14 days but there was no answer until after 2 more months I suddenly got access to the Student Pack again.

There's not much you can do other than wait, in my opinion. I wish you much success
